src/objects/media/media.scss

```diff
@@ -50,3 +52,49 @@
 .o-media__content {
   flex: 1;
 }
+
+/**
+ * We use CSS grid when supported by the browser.
+ */
+
+@supports (display: grid) {
+  .o-media {
+    align-items: center;
+    display: grid;
+    grid-gap: size.$spacing-media-gap;
+    grid-template-columns: minmax(0, auto) 1fr;
+  }
+
+  @for $i from 1 through 4 {
+    .o-media--1by#{$i} {
```

Comment:
I like this class name, but can we add hyphens so it's `--1-by-X` for consistency with our other class names?